src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostInitiateByIdControllerTest.java

```diff
@@ -757,11 +756,12 @@ void should_return_201_with_task_unassigned_when_invalid_old_assignee_auto_assig
                 jsonPath("$.task_id").value(taskId),
                 jsonPath("$.task_name").value("aTaskName"),
                 jsonPath("$.task_type").value("followUpOverdueReasonsForAppeal"),
-                jsonPath("$.state").value("UNASSIGNED"),
+                jsonPath("$.state").value("ASSIGNED"),
+                jsonPath("$.assignee").value("anotherAssignee"),
                 jsonPath("$.task_system").value("SELF"),
                 jsonPath("$.security_classification").value("PUBLIC"),
                 jsonPath("$.title").value("aTaskName"),
-                jsonPath("$.auto_assigned").value(false),
+                jsonPath("$.auto_assigned").value(true),
```

Comment:
I would expect the auto assignable to be false when the auto assignable flag is false for the role.

Reply:
reverted